### ISSUE TITLE
Waypoints 

### ISSUE THREAD

**Issue by Fulminao** [fhorinek/SkyDrop#153]
Hi, 

It would be very interesting and useful to insert, in some way, waypoints thats are very useful during a cross country fly, a competition, storing a position of a thermal, etc.
It will be possible?

Thanks.

**Comment by fhorinek** [comment]
Do you mean pre-flight path waypoints or defining waypoints during flight?

**Comment by Fulminao** [comment]
I mean pre-flight path waypoints. So that I can place them before the race.

2016-03-23 8:41 GMT+01:00 František Horínek <notifications@github.com>:

**Comment by fhorinek** [comment]
Adding to todo list

**Comment by Fulminao** [comment]
I have a file, with "wpt" extension,which I can send a old 2015 example
file to you in allegate that I must use for the race. It will be very
beautiful if I put directly this file into the Skydrop SD directly and the
Skydrop can read itself this file.

Thanks

2016-03-23 9:42 GMT+01:00 František Horínek <notifications@github.com>:

**Comment by fhorinek** [comment]
I would like to see the format.
my mail si frantisekhorinek@gmail.com

**Comment by mattcanning** [comment]
If this feature is implemented it would be useful to add a C-record to the IGC file as well, so that it's possible to have your flight declaration stored directly in the instrument.

**Comment by almonn** [comment]
I think its important to carefully manage feature requests on the Skydrop. 
If the functionality grows too much it will become unmanagable on such a small screen. 
Better in my opinion to keep the Skydrop to 'core/essential' functions, and when complex navigational/computational functions are required, simply feed the GPS & Barometric data to another device (via Bluetooth or Serial/UART) where (i) a more suitable/larger screen can be used, and (ii) the computations have already been achieved by a complex/mature product (e.g. XCSoar).

**Comment by bubeck** [comment]
Implemented as https://github.com/fhorinek/SkyDrop/pull/337

**Comment by AberelGo** [comment]
Hi! I try to navegate with skydrop to waypoints. I install the latest update 4016. The problem is my Skydrop no recognize my "waypoint.wpt" file (.WPT is a extension, not a txt file). I followed the instrucctions in this thread [https://github.com/fhorinek/SkyDrop/wiki/Navigation-and-aerodrome-files](url)  but... don't works. Please can you help me? Sorry for my bad English

**Comment by bubeck** [comment]
Did you put the waypoint.wpt file into a subdirectory called "WAYPOINT"?

What exactly is not working? 

  1.  You must first select the file by choosing "Settings>Navigation>Waypoint". Do you see your file in the list of available files or is the list empty? 
  2. Are you able to select the file and see the details (like the number of waypoints) in the details page?
  3. Did you add the waypoint widgets to a screen?
  4. Do the widgets show something?

**Comment by AberelGo** [comment]
Yes, I have two subdirectories in the memory. HOMES and WAYPOINT. And have a HOME widgets in one screen (the HOME functionallity works well). The HOMES widgets are the same for the navigation to waypoints? When I navigate for the menu "Settings>Navigation>Waypoint" found the file.WPT but the skydrop don't open it. This file only contains two lines:

WP MONT1 42.1089207 2.34940052 1000
WP MONT2 42.1251877 2.38785266 1000

Thanks for the reply bubeck!

**Comment by bubeck** [comment]
You _must_ use quotes around the name

    WP "MONT1" 42.1089207 2.34940052 1000
    WP "MONT2" 42.1251877 2.38785266 1000

**Comment by AberelGo** [comment]
Ok. I put a waypoint widgets in one screen. And put quotes around the name of the points. But when I select the file. The skydrop do nothing. I think is a problem with the format of the file

**Comment by bubeck** [comment]
Could you please send me the file exactly as it is as an attachment
